### DISCUSSION TITLE
On hexdocs pages, make it so when clicking on last function, it shows up at the top of the screen

### DISCUSSION THREAD

**Discussion by bentomas** [gleam-lang/gleam#3702] [Ideas & suggestions]
I'm constant clicking on the last function, and then its info is short and more than one function is visible, and then reading the documentation for the wrong function.  If we were to make the div for the last function in the list the same height as the window jumping to its anchor would position the div at the top of screen.  

we'd have to add this css to the last div to make it work, but also make it line up nicely when scrolling down manually:

```
min-height: calc(100vh - var(--large-gap) - var(--gap) - var(--header-height));
```

**Comment by sz3lbi** [comment]
I agree. I had a similar problem in the beginning.

**Comment by lpil** [comment]
Doesn't this add a large empty space to the bottom of the page?

**Reply by bentomas** [reply]
Yep, that's exactly what it does. You can try it out in a web console, just adding that css to the last div. I think the trade off is worth it, but that's just me.
